media-libs/vidstab/vidstab-1.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+MY_PN="vid.stab"
+DESCRIPTION="Video stabilization library"
+HOMEPAGE="http://public.hronopik.de/vid.stab/"
+SRC_URI="https://github.com/georgmartius/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Are you sure it's not GPL-2+?

Reply:
Yes indeed! Thanks for catching that, I missed the "version 2 , or later" part... will fix shortly.